recipes/pcaudiolib/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pcaudiolib" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/espeak-ng/{{ name }}/archive/c651ccb767abede0228570968293219e429899d5.zip
+  sha256: 37ad186d34acb9c609137e5a01b1467162e75248e3678052992421ebd712a58f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('alsa-lib-devel') }}  # [linux]
+    - cmake
+    - coreutils  # [osx]
+    - autoconf  # [not win]
+    - automake  # [not win]
+    - make  # [not win]
+    - ld_impl_linux-64  # [linux]
+    - libtool  # [not win]
+    - pkg-config  # [not win]
+    - alsa-lib  # [linux]
+    - lld  # [win]
+    - m2-autoconf  # [win]
+    - m2-automake-wrapper  # [win]
+    - m2-base  # [win]
+    - m2-findutils  # [win]
+    - m2-libtool  # [win]
+    - m2-make  # [win]
+    - m2-pkg-config  # [win]
+  host:
+    - alsa-lib  # [linux]
+  run:
+    - alsa-lib  # [linux]
+test:
+  commands:
+    - test -f $PREFIX/lib/libpcaudio.so  # [linux]
```

Comment:
Shared library packages need run_exports, so that downstream packages always depend on a compatible version.

What kind of ABI compatability guarantees and versioning does this project use? Is the ABI guaranteed to only break when the soname major version increases?

Reply:
As a packager of this library, you need to figure it out. Read about conda run_exports and ask the creators of this package what they plan for ABI compatability.